src/Auth0Client.ts

```diff
@@ -51,6 +51,7 @@ import {
   RedirectLoginResult,
   GetTokenSilentlyOptions,
   GetTokenWithPopupOptions,
+  IdToken,
```

Comment:
Adding this results in an unused import.
I think we should explicitly add the return type to `getIdTokenClaims`:

```
public async getIdTokenClaims(options: GetIdTokenClaimsOptions = {}): Promise<IdToken>
```

Reply:
Thanks - yes, I saw the build bot mentioned that - I've removed the unused reference, and ensured that the correct type is returned from the func.   This fixes the issue within the TYPINGS file...